src/components/Search.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from './Button'
+
+//require('dotenv').config()
+const API_KEY = process.env.REACT_APP_API_KEY
+
+export default function Search({ onSubmit }) {
+  const [options, setOptions] = useState([])
+  const [query, setQuery] = useState('')
+  const [selectedOption, setSelectedOption] = useState(null)
+  useEffect(() => {
+    setSelectedOption(null)
+    fetch(`https://api.rawg.io/api/games?key=${API_KEY}&search=${query}`)
+      .then(res => res.json())
+      .then(data => setOptions(data.results))
```

Comment:
You might want to add a `.catch` and in a future issue not logging it out but providing an error message on the screen.
```suggestion
    fetch(`https://api.rawg.io/api/games?key=${API_KEY}&search=${query}`)
      .then(res => res.json())
      .then(data => setOptions(data.results))
      .catch(error => console.error(error))
```